ocs_ci/utility/vsphere.py

```diff
@@ -468,35 +468,24 @@ def destroy_pool(self, pool, dc, cluster):
         WaitForTask(pi.Destroy())
         logger.info(f"Successfully deleted resource pool {pool}")
 
-    def remove_disk(self, vm, unit_number, datastore=True):
+    def remove_disk(self, vm, identifier, key='unit_number', datastore=True):
```

Comment:
the signature changed, please confirm no calls with old one

Reply:
correct. The only additional call was adjusted accordingly:
https://github.com/red-hat-storage/ocs-ci/pull/1245/files#diff-2c4f7dc9d7d80db5ddfb8abfbfbce05aR544